app/src/main/res/xml/feed_settings.xml

```diff
@@ -18,6 +18,14 @@
             android:title="@string/auto_delete_label"
             android:key="autoDelete"/>
 
+    <ListPreference
+            android:entries="@array/spnVolumeReductionItems"
+            android:entryValues="@array/spnVolumeReductionValues"
+            android:summary="Turn down volume for episodes of this feed:  \%s"
```

Comment:
This should be a `@string` resource. Also, please add only one space between description and value.

Reply:
Done - the double space seemed to make the current value more obvious, but feels wrong, admittedly :)